ios/Sources/AppFeature/AppScreen.swift

```diff
@@ -0,0 +1,45 @@
+import AboutFeature
+import FavoritesFeature
+import HomeFeature
+import MediaFeature
+import SwiftUI
+
+public struct AppScreen: View {
+    @State var selection = 0
+
+    public init() {}
+
+    public var body: some View {
+        TabView(
+            selection: $selection,
+            content: {
+                HomeScreen()
```

Comment:
It seems better to use any router or coordinator classes to call views.

Reply:
It's good!
However I think this is overdoing in this PR. So I will make other GitHub issues. 🙏🏼